src/Nethermind/Nethermind.Logging.NLog/NLogLogger.cs

```diff
@@ -2,16 +2,20 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 
 using System;
-using System.IO;
-using System.Linq;
-using System.Runtime.CompilerServices;
+using System.Collections.Concurrent;
+using System.Threading;
+
 using NLog;
-using NLog.Targets;
+
+#nullable enable
 
 namespace Nethermind.Logging.NLog
 {
-    public class NLogLogger : ILogger
+    public class NLogLogger : ILogger, IThreadPoolWorkItem
     {
+        private readonly ConcurrentQueue<LogEntry> _workItems = new();
```

Comment:
Now every of our thousands logger has a ConcurrentQueue? Wouldn't it affect performance and memory? Wouldn't it be better to have just one queue per application?

Reply:
Changed to single static queue